solutionbox/structured_data/mltoolbox/_structured_data/preprocess/local_preprocess.py

```diff
@@ -135,7 +136,12 @@ def _init_numerical_results():
       json.dumps(numerical_results, indent=2, separators=(',', ': ')))
 
   # Write the vocab files. Each label is on its own line.
-  for name, unique_labels in categorical_results.iteritems():
+  if sys.version_info.major > 2:
+    name_label_itr = categorical_results.items()
+  else:
+    name_label_itr = categorical_results.iteritems()
+
+  for name, unique_labels in name_label_itr:
```

Comment:
See my other comment about potentially having a compatibility function so that this could be changed to:

```
  for name, unique_labels in compat_iteritems(categorical_results):
```